### PR TITLE
update formatting for godoc generation

### PR DESCRIPTION
**Pull request by thinkingserious**
Internal Reference: DI-1132

These changes allow for better formatting using gdoc. (example [here](https://github.com/twilio/twilio-go/tree/godocs/localhost:8080/pkg/github.com/twilio))

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/main/resources/twilio-go/api.mustache [hunk 1]
```diff
@@ -38,26 +38,29 @@ type {{{nickname}}}Params struct {
...
 {{#notes}}
-* {{notes}}
+// {{notes}}
+//
 {{/notes}}
 {{#allParams}}
 {{#required}}
-* @param {{paramName}}{{#description}} {{{.}}}{{/description}}
+// param: {{paramName}}{{#description}} {{{.}}}{{/description}}
+//
```
Comment (shwetha-manvinkurke) [inline src/main/resources/twilio-go/api.mustache:50]: Is this empty line needed?

Comment (thinkingserious) [reply src/main/resources/twilio-go/api.mustache:50]: Yes, it functions as a line break for godocs generation. [This is an example](https://pkg.go.dev/github.com/twilio/twilio-go@v0.6.0/twilio/rest/accounts/v1#DefaultApiService.CreateCredentialAws) of formatting without the line breaks.

Comment (thinkingserious) [reply src/main/resources/twilio-go/api.mustache:50]: [Here is a pointer](https://stackoverflow.com/a/51641739) to the [blog post](https://blog.golang.org/godoc) that documents this functionality.

Comment (shwetha-manvinkurke) [reply src/main/resources/twilio-go/api.mustache:50]: Hmm I don't like that go does that :unamused:

**Review by shwetha-manvinkurke — APPROVED** [review]

**Pull request merged**